src/utils/validation.js

```diff
@@ -10,13 +10,19 @@ const {combineRequestSchemas} = require('./combine_request_schemas');
  * uses ajv to validate request parameters against schema determined by request route and request method
  * $ref will resolve only `#/components/schemas/...` same as openapi schema
  */
-module.exports = ({components, paths, ajvOptions, ajvKeywords = []}) => {
+module.exports = ({components, paths, ajvOptions = {}, ajvKeywords = [], ajvErrorsOptions}) => {
```

Comment:
`ajvErrorsOptions = {}` also?

Reply:
it's not necessary , but for consistent, It is better {} also. I will fix it now.